src/server/system_services/schemas/nsfs_account_schema.js

```diff
@@ -60,6 +60,13 @@ module.exports = {
                     encrypted_secret_key: {
                         type: 'string',
                     },
+                    creation_date: {
+                        type: 'string',
+                    },
+                    // we would need to translate the is_active to 'Active' and 'Inactive' strings
+                    is_active: {
+                        type: 'boolean',
+                    },
```

Comment:
since we consider this property being absent to mean it is implicitly active, I would prefer this prop to have the opposite meaning - such as `disabled` or `deactivated`.